src/v/cloud_storage/async_manifest_view.cc

```diff
@@ -505,30 +503,44 @@ async_manifest_view::get_active(async_view_search_query_t query) noexcept {
         // This calls 'get_materialized_manifest' internally which
         // could potentially schedule manifest hydration/materialization
         // in the background fiber.
-        auto result = co_await cursor->seek(query);
-        if (result.has_error()) {
-            vlog(
-              _ctxlog.error,
-              "failed to seek to offset {}, error: {}",
-              query,
-              result.error());
-            co_return result.as_failure();
-        }
-        if (!result.value()) {
-            vlog(
-              _ctxlog.debug,
-              "failed to seek to offset {}, offset out of valid range",
-              query);
-            co_return error_outcome::out_of_range;
+        int retry_quota = 4;
```

Comment:
We should remove the previously inlined retries from `remote_partition::init_cursor` now

Reply:
done